$OpenBSD: patch-gdevilspie,v 1.1 2011/11/11 15:57:40 jasper Exp $

OpenBSD lacks the Python WNCK module after py-gnome-desktop was removed.

--- gdevilspie.orig	Sat Jun 12 03:41:19 2010
+++ gdevilspie	Fri Nov 11 16:56:58 2011
@@ -36,14 +36,7 @@ except:
 	print "pyGTK is not correctly installed, exiting."
 	sys.exit(1)
 
-DISABLE_FILLER=""
-try:
-	import gDevilspie.filler as filler
-except:
-	error = gtk.MessageDialog(None, 0, gtk.MESSAGE_ERROR, gtk.BUTTONS_CLOSE, "Make sure that Python-Wnck is correctly installed for 'filler' module to run properly.")
-	response = error.run()
-	error.destroy()
-	DISABLE_FILLER="True"
+DISABLE_FILLER="True"
 
 DISABLE_XDG=""
 try:
